.travis.yml

```diff
@@ -96,12 +96,17 @@ script:
   - ls ${TRAVIS_BUILD_DIR}/wheelhouse/ || echo "no wheelhouse"
 
 deploy:
-  provider: script
-  skip_cleanup: true
-  script: scripts/deploy.sh
-  on:
-    branch: master
-    tags: true
+  - provider: script
```

Comment:
This should only run on tagged releases. Otherwise you will add the release multiple times to the website.

Reply:
That is handled by
```yaml
on:
  tags: true
```
or at least it should be.

Reply:
Uhps, my bad :/